docs/reference/ilm/ilm-skip-rollover.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[skipping-rollover]]
+== Skip rollover
+
+When `index.lifecycle.indexing_complete` is set to `true`, 
+{ilm-init} won't perform the rollover action on an index, 
+even if it otherwise meets the rollover criteria. 
+It's set automatically by {ilm-init} when the rollover action completes successfully.
+
+You can set it manually to skip rollover if you need to make an exception
+to your normal lifecycle policy and update the alias to force a roll over, 
+but want {ilm-init} to continue to manage the index.
+
+If an index's lifecycle policy is removed, this setting is also removed.
+
+IMPORTANT: When `index.lifecycle.indexing_complete` is `true`, 
+{ilm-init} verifies that the index is no longer the write index 
+for the alias specified by `index.lifecycle.rollover_alias`.
+If the index is still the write index or the rollover alias is not set,
+the index is moved to the <<index-lifecycle-error-handling,`ERROR` step>>.
+
+For example, if you need to change the name of new indices in a series while retaining
+previously-indexed data in accordance with your configured policy, you can:
+
+. Create a template for the new index pattern that uses the same policy. 
+. Bootstrap the initial index.
+. Change the write index for the alias to the bootstrapped index 
+using the <<indices-aliases, index aliases>> API.
+. Set `index.lifecycle.indexing_complete` to `true` on the old index to indicate 
+that it does not need to be rolled over.
+
+{ilm} continues to manage the old index in accordance with your existing policy. 
```

Comment:
```suggestion
{ilm-init} continues to manage the old index in accordance with your existing policy. 
```

Reply:
The `{ilm}` variable renders as "index lifecycle management", which doesn't work at the beginning of the sentence:
>index lifecycle management continues...